BO4E/BO/Tranche.cs

```diff
@@ -0,0 +1,63 @@
+using BO4E.COM;
+using BO4E.EnergyIdentificationCodes;
+using BO4E.ENUM;
+using BO4E.meta;
+
+using Newtonsoft.Json;
+
+using ProtoBuf;
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.ConstrainedExecution;
+using System.Text.Json.Serialization;
+using System.Text.RegularExpressions;
+
+namespace BO4E.BO
+{
+    /// <summary>
+    ///     Objekt zur Aufnahme der Informationen zu einer Tranche
+    /// </summary>
+    [ProtoContract]
+    public class Tranche : BusinessObject
+    {
+        /// <summary>
+        ///     Identifikationsnummer einer Tranche, an der Energie entweder
+        ///     verbraucht, oder erzeugt wird (Like MarktlokationsId <see cref="Marktlokation"/>)
+        /// </summary>
+        [DefaultValue("|null|")]
+        [JsonProperty(Required = Required.Always, Order = 10, PropertyName = "trancheId")]
+        [JsonPropertyName("trancheId")]
+        [JsonPropertyOrder(10)]
+        [DataCategory(DataCategory.POD)]
+        [BoKey]
+        [ProtoMember(4)]
+        public string TrancheId { get; set; }
+
+        /// <summary>Sparte der Messlokation, z.B. Gas oder Strom.</summary>
```

Comment:
Copy-Paste-Rückstände ;) 
```suggestion
        /// <summary>Sparte der Tranche, z.B. Gas oder Strom.</summary>
```